core/templates/dev/head/domain/exploration/EditableExplorationBackendApiServiceSpec.js

```diff
@@ -0,0 +1,249 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for EditableExplorationBackendApiService.
+ */
+
+describe('Editable exploration backend API service', function() {
+  var EditableExplorationBackendApiService = null;
+  var ReadOnlyExplorationBackendApiService = null;
+  var sampleDataResults = null;
+  var $rootScope = null;
+  var $scope = null;
+  var $httpBackend = null;
+  var UndoRedoService = null;
+
+  beforeEach(module('oppia'));
+  beforeEach(module('oppia', GLOBALS.TRANSLATOR_PROVIDER_FOR_TESTS));
+
+  beforeEach(inject(function($injector) {
+    EditableExplorationBackendApiService = $injector.get(
+      'EditableExplorationBackendApiService');
+    ReadOnlyExplorationBackendApiService = $injector.get(
+      'ReadOnlyExplorationBackendApiService');
+    UndoRedoService = $injector.get('UndoRedoService');
```

Comment:
No we don't, I believe I was using them in an earlier iteration of a test and never took it out. Removed.